src/buttons/docs/readme.md

```diff
@@ -44,3 +44,7 @@ With the buttons directive, we can make a group of buttons behave like a set of
 * `toggleEvent`
   _(Default: `click`)_ -
   Event used to toggle the buttons.
+
+### Known issues
+
+To use tooltips or popovers on a button group, make sure you add the applicable tooltip or popover directive directly to your `btn-group` element and set `appendToBody` to `true`. This is due to Bootstrap CSS styling. See [here](http://getbootstrap.com/components/#btn-groups) for more information.
```

Comment:
The append-to-body is only necessary if adding the tooltip to an element inside the btn-group.